src/components/v5/common/CompletedAction/partials/MultiSigMeatballMenu/MultiSigMeatballMenu.tsx

```diff
@@ -1,88 +1,27 @@
-import { Copy, Prohibit } from '@phosphor-icons/react';
+import { Copy } from '@phosphor-icons/react';
 import React, { type FC } from 'react';
 import { generatePath } from 'react-router-dom';
 
 import MeatballMenuCopyItem from '~common/ColonyActionsTable/partials/MeatballMenuCopyItem/MeatballMenuCopyItem.tsx';
 import { APP_URL } from '~constants';
 import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
-import { useCompletedActionContext } from '~context/CompletedActionContext/CompletedActionContext.ts';
-import { MultiSigVote } from '~gql';
-import useAsyncFunction from '~hooks/useAsyncFunction.ts';
-import { ActionTypes } from '~redux';
-import { type VoteOnMultiSigActionPayload } from '~redux/sagas/multiSig/voteOnMultiSig.ts';
 import {
   COLONY_ACTIVITY_ROUTE,
   COLONY_HOME_ROUTE,
   TX_SEARCH_PARAM,
 } from '~routes';
-import { extractColonyRoles } from '~utils/colonyRoles.ts';
-import { extractColonyDomains } from '~utils/domains.ts';
 import { formatText } from '~utils/intl.ts';
-import { getMultiSigRequiredRole } from '~utils/multiSig.ts';
 import MeatBallMenu from '~v5/shared/MeatBallMenu/index.ts';
 import { type MeatBallMenuItem } from '~v5/shared/MeatBallMenu/types.ts';
 
 import { type MultiSigMeatballMenuProps } from './types.ts';
 
 const displayName = 'v5.common.CompletedAction.partials.MultiSigMeatballMenu';
 
-const MULTISIG_EXPIRY_PERIOD = 604_800_000; // 1 week
-
 const MultiSigMeatballMenu: FC<MultiSigMeatballMenuProps> = ({
   transactionHash,
-  multiSigData,
-  isOwner,
-  actionType,
 }) => {
   const { colony } = useColonyContext();
-  const { showRejectMultiSigStep, setShowRejectMultiSigStep } =
-    useCompletedActionContext();
```

Comment:
you can actually also delete this context and its usage if you want some negative line count to your name :D